spec/requests/users_spec.rb

```diff
@@ -14,6 +14,21 @@
     expect(json_body[1]['id']).to eq(other.id)
     expect(json_body[1]['name']).to eq(other.name)
   end
+
+  it 'should return skateparks favorites by users' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.